File: comms/push-notifications/credentials/update-credential/update-credential.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;
import com.twilio.sdk.resources.pushnotifications.credentials.requests.CredentialsUpdateRequest;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("<username>", "<password>")
            .build();

        client.pushNotifications().credentials().update(
            "comms_credential_01h9krwprkeee8fzqspvwy6nq8",
            CredentialsUpdateRequest
                .builder()
                .isDefault(true)
                .build()
        );
    }
}

File: comms/push-notifications/credentials/update-credential/update-credential.cs
using Twilio.Comms;
using System.Threading.Tasks;
using Twilio.Comms.PushNotifications;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.PushNotifications.Credentials.UpdateAsync(
            "comms_credential_01h9krwprkeee8fzqspvwy6nq8",
            new CredentialsUpdateRequest {
                IsDefault = true
            }
        );
    }

}
